tests/test_operators.py

```diff
@@ -2028,6 +2028,55 @@ def test_has_same_values(self):
         result = DataframeType({"value": invalid_df, }).has_same_values({"target": "MHCAT", })
         self.assertTrue(result.equals(pandas.Series([True, True, True, True, True, ])))
 
+    def test_is_ordered_by(self):
```

Comment:
Can we add a test for dates and strings?

Reply:
Added in latest commit